releases/release-4.0.16.md

```diff
@@ -0,0 +1,109 @@
+---
+title: TiDB 4.0.16 Release Notes
+---
+
+# TiDB 4.0.16 Release Notes
+
+Release Date: December 10, 2021
+
+TiDB version: 4.0.16
+
+## Compatibility changes
+
++ TiKV
+
+    - When casting an invalid utf8 string to real, try to truncate the utf8 prefix instead of returning an error [#11466](https://github.com/tikv/tikv/issues/11466)
+
++ Tools
+
+    + TiCDC
+
+        - Change the default value of Kafka Sink `max-message-bytes` to 1 MB [#2962](https://github.com/pingcap/ticdc/issues/2962)
+        - Change the default value of Kafka Sink `partition-num` to 3 [#3337](https://github.com/pingcap/ticdc/issues/3337)
+
+## Improvements
+
++ TiKV
+
+    - sst_importer: Reduce the space usage when using BR-Restore or Lightning-Local-backend by using zstd compression in SSTs  [#11469](https://github.com/tikv/tikv/issues/11469)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - Improve the robustness for restoring [#27421](https://github.com/pingcap/tidb/issues/27421)
+
+    + TiCDC
+
+        - Add a rate limiter to limit the EtcdWorker tick frequency [#3112](https://github.com/pingcap/ticdc/issues/3112)
```

Comment:
```suggestion
        - Add a Tick frequency limit to EtcdWorker to prevent pd's Etcd writes from affecting pd services too many times [#3112](https://github.com/pingcap/ticdc/issues/3112)
```